File: dom-frontend/dom-front/src/app/model/StudentDTO.ts
export class StudentDTO {
  id?: number;
  username?: string;
  password?: string;
  ime?: string;
  prezime?: string;
  jmbg?: string;
  godinaStudiranja?: number;
  osvojeniBodovi?: number;
  prosek?: number;
  kartica?: string; // Ako je kartica enum, trebaš koristiti odgovarajući tip
  bodovi?: number;
  konkursId?: number; // Ako backend koristi Long, možeš ostaviti kao number ako frontend ne koristi Long
  sobaId?: number; // Ako backend koristi Long, možeš ostaviti kao number ako frontend ne koristi Long
}

File: auth-service/src/main/java/com/ftn/authservice/authservice/entity/StudentDTO.java
package com.ftn.authservice.authservice.entity;

import lombok.Data;

@Data
public class StudentDTO {
    private Long id;
    private String username;
    private String password;
    private String ime;
    private String prezime;
    private String jmbg;
    private int godinaStudiranja;
    private int osvojeniBodovi;
    private double prosek;
    private Kartica kartica;
    private double bodovi;
    private Long konkursId;
    private Long sobaId;


}
